Makefile

```
@@ -1,9 +1,10 @@
 NCOMPUTES ?= 2
+PXEBOOT_ISO=
 
 cluster:
```

Could we have this depend on a new target?
```suggestion
$(PXEBOOT_ISO):
    wget -O "$@" http://boot.ipxe.org/ipxe.iso
    
cluster: $(PXEBOOT_ISO)
```